File: WdtcCore/src/test/java/org/wdt/wdtc/core/test/URLTest.java
package org.wdt.wdtc.core.test;

import org.junit.jupiter.api.Test;
import org.wdt.wdtc.core.utils.URLUtils;

import java.io.IOException;

public class URLTest {
    @Test
    public void getRedirectUrlTest() throws IOException {
        System.out.println(URLUtils.isNetworkHasThisFile(URLUtils.toURL("https://bmclapi2.bangbang93.com/version/1.20.1/json")));
    }
}

File: WdtcCore/src/test/kotlin/org/wdt/wdtc/core/test/URLTest.kt
package org.wdt.wdtc.core.test

import org.wdt.wdtc.core.utils.toURL
import java.net.URL
import kotlin.test.Test

class URLTest {
  @Test
  fun testUrlbuilder() {
    println(URL("https://www.jetbrains.com.cn/".toURL(), "idea/download"))
  }
}
